DESCRIPTION

```diff
@@ -2,26 +2,25 @@ Package: lockbox
 Type: Package
 Title: lockbox (http://github.com/robertzk/lockbox)
 Description: Bundler-style dependency management for R.
-Version: 0.1.12
+Version: 0.1.13
```

Comment:
:+1:

I think it's 0.2 per [semver](http://semver.org/) since the change is backwards compatible.  Maybe 0.2.9000 until we know it's stable?

I think it will become 1.0 if/when we try to get it on CRAN. That's what Hadley seems to do, anyway.